### ISSUE TITLE
Update inconsistent text in analyzer for Null-conditional Assignment

### ISSUE THREAD

**Issue by KathleenDollard** [dotnet/roslyn#77985]
**Brief description:**

The text for rule CS0031 is lovely "Null check can be simplified"

Unfortunately, the fixer text is "Use null propagation"  which does not align with the rule name and also differs from how we refer to the feature ("Null-conditional assignment")

It takes a few extra brain cycles to select the correct fixer when it differs from the name of the rule. 

"Simplify null check" would align and what I think would be best. "Use null-conditional assignment" would align with the feature name as a second choice.

**Languages applicable:**

C#

**Code example that the analyzer should report:**

This issue regards the text when you hover on `if` in the following code:

```csharp
    public static void UpdateAge(Customer? customer, int newAge)
    {
        if (customer is not null)
        {
            customer.Age = newAge;
        }
    }
```

**Comment by CyrusNajmabadi** [comment]
This has been the text on this feature since it was introduced in 2016.  It's widely used, and hasn't seemed to be a problem so far.  I would prefer we just keep it as is.

**Comment by CyrusNajmabadi** [comment]
Closing as not planned.
